src/modules/shared/pages/innovation/sections/section-summary.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, Input, OnChanges, OnInit, SimpleChanges } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { CoreComponent } from '@app/base';
+import { ContextInnovationType } from '@app/base/types';
+
+import { WizardEngineModel, WizardSummaryType } from '@modules/shared/forms';
+import { InnovationDocumentsListOutDTO } from '@modules/shared/services/innovation-documents.service';
+import { INNOVATION_SECTION_STATUS, InnovationStatusEnum } from '@modules/stores/innovation';
+import { innovationSectionsWithFiles } from '@modules/stores/innovation/innovation-record/ir-versions.config';
+import { InnovationSectionStepLabels, } from '@modules/stores/innovation/innovation-record/ir-versions.types';
+
+@Component({
+    selector: 'shared-innovation-summary',
+    templateUrl: './section-summary.component.html'
+})
+export class InnovationSectionSummaryComponent extends CoreComponent implements OnInit {
+
+  @Input() sectionInfo: {
+    id: string,
+    nextSectionId: null | string,
+    title: string,
+    status: { id: keyof typeof INNOVATION_SECTION_STATUS, label: string },
+    submitButton: { show: boolean, label: string },
+    isNotStarted: boolean,
+    hasEvidences: boolean,
+    wizard: WizardEngineModel,
+    allStepsList: InnovationSectionStepLabels,
+    date: string,
+    submittedBy: null | { name: string, isOwner?: boolean },
+    openTasksCount: number
+  };
```

Comment:
I think this @Input could be much smaller, there are items here that are not needed for this component.